composer.json

```diff
@@ -26,7 +26,8 @@
     ],
     "require": {
         "yiisoft/yii2": "~2.0.0",
-        "psy/psysh": "~0.8.1"
+        "psy/psysh": "~0.8.1",
+        "symfony/var-dumper": "*"
```

Comment:
I think we need to specify a version there.

Reply:
Like so `2.* || 3.* || 4.*`?

Reply:
~2.7|~3.0|~4.0